src/NzbDrone.Core/Download/Clients/Tribler/Models/TriblerDownloadClientApi.cs

```diff
@@ -0,0 +1,257 @@
+namespace NzbDrone.Core.Download.Clients.Tribler
+{
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.5.2.0 (Newtonsoft.Json v11.0.0.0)")]
+    public enum DownloadStatus
```

Comment:
Auto generated?

Reply:
Would you like me to remove the GeneratedCode annotations?
They are leftover from when i started by generating from a swagger.json

Reply:
I removed the annotations, as the do not make any sense anymore.